app/javascript/hooks/use-tooltip-content-query.ts

```diff
@@ -0,0 +1,46 @@
+import { useQuery } from 'react-query'
+
+function fetchContent(endpoint: string, signal: AbortSignal) {
+  return fetch(endpoint, {
+    method: 'get',
+    signal,
+  }).then((res) => {
+    if (res.status != 200 && res.status != 304) {
```

Comment:
Is this good by others to have here? So that it only tries to display to tool tip on 200 (success) or 304 (cached)?

Reply:
My Ruby brain would rather group the statuses:

```
if(![200,304].include?(res.status))
```

Maybe even pull them into "validStatuses" and add the comment about `200 = success and 304 = cached=` in the code for future reference.

But that might be less good JS 🤷

Reply:
```suggestion
    if (![200, 304].includes(res.status)) {
```

Reply:
They're equally fine!